lib/ios-crash-log.js

```diff
@@ -0,0 +1,61 @@
+import path from 'path';
+import _ from 'lodash';
+import _glob from 'glob';
+import { fs, getLogger } from 'appium-support';
+import B from 'bluebird';
+import { asyncmap } from 'asyncbox';
+
+const glob = B.promisify(_glob);
+
+
+require('date-utils');
```

Comment:
I think it should be moved to the other file, as well, since (a) we already have a comment there, and (b) we seem to actually use it there (I don't see any of the monkeypatched `Date` methods in use in this file).